applications/RANSApplication/custom_elements/incompressible_potential_flow/laplace_element.h

```diff
@@ -0,0 +1,500 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_LAPLACE_ELEMENT_H_INCLUDED)
+#define KRATOS_LAPLACE_ELEMENT_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/checks.h"
+#include "includes/define.h"
+#include "includes/element.h"
+
+// Application includes
+#include "custom_utilities/rans_calculation_utilities.h"
+
+namespace Kratos
+{
+///@name Kratos Classes
+///@{
+
+template <unsigned int TDim, unsigned int TNumNodes>
+class LaplaceElement : public Element
```

Comment:
We should probably move this `Laplacian` to the `elements` in the `KratosCore`. @KratosMultiphysics/technical-committee

Reply:
(OFC for a future standalone PR)

Reply:
:+1: